diff --git a/CHANGELOG.md b/CHANGELOG.md
deleted file mode 100644
index 189813027867b0ec0843e1893056d4ae137ffac3..0000000000000000000000000000000000000000
diff --git a/index.js b/index.js
index d5903b96842bf9e4019b5cd8070baeaa209d8aef..add26f98889ae57cd071cd680b0371c88aa9a2d6 100644
--- a/index.js
+++ b/index.js
@@ -1,5 +1,6 @@
 import stylelint from 'stylelint';
 import {showInvisibles, generateDifferences} from 'prettier-linter-helpers';
+import fs from 'fs';
 
 const prettierPromise = import('prettier');
 
@@ -60,41 +61,10 @@ const ruleFunction = (expectation, options, context) => {
       return;
     }
 
-    const initialOptions = {};
-
-    // If no filepath was provided then assume the CSS parser
-    // This is added to the options first, so that
-    // prettierRcOptions and stylelintPrettierOptions can still override
-    // the parser.
-    if (filepath == '<input>') {
-      initialOptions.parser = 'css';
-    }
-
-    // Stylelint supports languages that may contain multiple types of style
-    // languages, thus we can't rely on guessing the parser based off the
-    // filename.
-
-    // In all of the following cases stylelint extracts a part of a file to
-    // be formatted and there exists a prettier parser for the whole file.
-    // If you're interested in prettier you'll want a fully formatted file so
-    // you're about to run prettier over the whole file anyway.
-    // Therefore running prettier over just the style section is wasteful, so
-    // skip it.
-
-    const parserBlockList = [
-      'babel',
-      'flow',
-      'typescript',
-      'vue',
-      'markdown',
-      'html',
-      'angular', // .component.html files
-      'svelte',
-      'astro',
-    ];
-    if (parserBlockList.indexOf(prettierFileInfo.inferredParser) !== -1) {
-      return;
-    }
+    // Enforce the parser is set to 'css'
+    const initialOptions = {
+      parser: 'css'
+    };
 
     const prettierOptions = Object.assign(
       {},
@@ -106,9 +76,11 @@ const ruleFunction = (expectation, options, context) => {
 
     let prettierSource;
 
-    const source = root.toString(result.opts.syntax);
+    let source = root.toString(result.opts.syntax);
+    console.warn( `source\n${source}`)
     try {
       prettierSource = await prettier.format(source, prettierOptions);
+      console.warn(`prettierSource\n${prettierSource}`)
     } catch (err) {
       if (!(err instanceof SyntaxError)) {
         throw err;
